### PR TITLE
Issue40  - YT videos playing without giving responding to the Cookie consent Manager 

### LINKED ISSUE #40: YT Videos should not play without accepting the Consent Manager

### PR DESCRIPTION
**Pull request by meejain**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #40 

Test URLs:
- Original: https://main--sunstar--sunstar-global.hlx.page/lets-talk-oral-health/episode-1-draft
- Before: https://main--sunstar-foundation--sunstar-foundation.hlx.page/_drafts/meet/episode-1-draft
- After: https://issue40-yt-consent--sunstar-foundation--sunstar-foundation.hlx.page/_drafts/meet/episode-1-draft

- [ ] New Blocks introduced in this PR
      If yes, please provide details below

Block Name    | Documentation
------------- | -------------
 For e.g. cards | [doc-link](https://main--sunstar-foundation--sunstar-foundation.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=0)

- [ ] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------
 For e.g. cards (grid)  | [doc-link](https://main--sunstar-foundation--sunstar-foundation.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=2)

### CODE REVIEW

**Comment by jindaliiita** [comment]
@meejain i am wondering why we need a new block here.
we can do changes in existing embed block regarding this. WDYT?
Also, with this change of making the YouTube video unclickable, I'm not sure if the [INP](https://web.dev/articles/inp), which will be part of the LHS on 12th March, will be affected or not.

**Comment by meejain** [comment]
> @meejain i am wondering why we need a new block here. we can do changes in existing embed block regarding this. WDYT? Also, with this change of making the YouTube video unclickable, I'm not sure if the [INP](https://web.dev/articles/inp), which will be part of the LHS on 12th March, will be affected or not.

Yes, I did give a try with embed - https://issue40-yt-consent--sunstar-foundation--sunstar-foundation.hlx.page/_drafts/meet/episode-1-draft-v1 

But giving us a wrong behaviour when click on Play button, need to refactor that block a bit it seems. Plus checking further with Engineering the impact of this logic on INP. Currently LHS is at 98.

File: blocks/yt-videos/yt-videos.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      anchor.classList.add('video-modal');
+      anchor.appendChild(picture);
+
+      // add unclickable behaviour to the anchor
+      anchor.classList.add('unclickable');
+
+      // Remove unclickable behaviour to the anchor after 3 seconds
+      setTimeout(() => {
+        anchor.classList.remove('unclickable');
+      }, 3500);
```
Comment (sdmcraft) [inline blocks/yt-videos/yt-videos.js:22]: Is there a way we can be more intelligent about it remove this class once the cookie consent has already been loaded.

**Comment by meejain** [comment]
Another way to fix the issue with another Pull request - https://github.com/sunstar-foundation/sunstar-foundation/pull/51. 

No need to - 

1) First have a word file with embed block under fragments/modals/videos. 
2) And then placing the URL & image int another block - YT-Videos

Simply enter the YT Video and the image under embed block - thats it.

**Comment by meejain** [comment]
Closing this Pull Request since the resolution is there via https://github.com/sunstar-foundation/sunstar-foundation/pull/51.

**Pull request closed without merging**